pipeline/outputs/stackdriver.md

```diff
@@ -16,7 +16,12 @@ Before to get started with the plugin configuration, make sure to obtain the pro
 | service\_account\_email | Account email associated to the service. Only available if **no credentials file** has been provided. | Value of environment variable _$SERVICE\_ACCOUNT\_EMAIL_ |
 | service\_account\_secret | Private key content associated with the service account. Only available if **no credentials file** has been provided. | Value of environment variable _$SERVICE\_ACCOUNT\_SECRET_ |
 | metadata\_server | Prefix for a metadata server. Can also set environment variable _$METADATA\_SERVER_.  | http://metadata.google.internal |
-| resource | Set resource type of data. Supported resource types: _k8s\_container_, _k8s\_node_, _k8s\_pod_, _global_ and _gce\_instance_. | global, gce\_instance |
+| resource | Set resource type of data. Supported resource types: _k8s\_container_, _k8s\_node_, _k8s\_pod_, _global_, _generic\_node_, _generic\_task_, and _gce\_instance_. | global, gce\_instance |
+| location | The GCP or AWS region in which to store data about the resource. Used for _generic\_node_ and _generic\_task_.|  |
+| namespace | A namespace identifier, such as a cluster name or environment. Used for _generic\_node_ and _generic\_task_.|  |
```

Comment:
Do you want to match the language in the description for `k8s\_cluster\_name`, etc, i.e.: `If the resource type is one of _generic\_node_ or _generic\_task_, then this field is required.`? Also for other fields.

Reply:
Yeah, Fixed.